experimental/storage/src/upload/types.ts

```diff
@@ -0,0 +1,68 @@
+import { PutObjectCommandInput, S3, S3Client, Tag as Tag_ } from "@aws-sdk/client-s3";
+
+export namespace Upload {
```

Comment:
nit: According to the lint, ES2015 `module` is preferred over Typescript namespace here.